src/test/java/com/scottlogic/SortingTests/AuthorSortingTests.java

```diff
@@ -0,0 +1,72 @@
+package com.scottlogic.SortingTests;
+
+import com.scottlogic.SortContext;
+import com.scottlogic.SortOrder;
+import com.scottlogic.SortingTests.utils.UserPostSortData;
+import com.scottlogic.UserPost;
+import com.scottlogic.sort.SortAuthor;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+
+public class AuthorSortingTests {
+
+    SortContext sortAuthor = new SortContext(new SortAuthor());
+
+    @Test
+    @DisplayName("Ascending functionality")
+    void sortAuthorAscending() {
+        List<UserPost> postsToSort = UserPostSortData.getUnsortedList();
+        List<UserPost> expectedAuthorSort = UserPostSortData.getSortedAuthorAscendingList();
```

Comment:
I feel like the expected data should be in the class or if it is shared by tests then be in a test utils class.
I understand that this is in your main code right now because it is a placeholder before you get actual data from some repository, but it is worth outlining the difference between placeholder data and test data.

Reply:
Is it not currently in a test utils class? I initially had the test data in each test class above the tests, but there was so much repetition that I figured it would be best to have the dummy data in it's own class, changing the method once rather than 5 times for each test, should I want to add another userPost/change a small detail/test for something new

Reply:
That's good, I thought it wasn't a test util. Thanks for clarifying.